CredentialsManager/FilesExceptions/DuplicateUserCredentialsException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace CredentialsManager.FilesExceptions
+{
+#pragma warning disable RCS1194 // Implement exception constructors.
+
+  public class DuplicateUserCredentialsException : Exception
+#pragma warning restore RCS1194 // Implement exception constructors.
+  {
+    public DuplicateUserCredentialsException()
+    {
+    }
+
+    public DuplicateUserCredentialsException(string? details) : base(details)
```

Comment:
Why `string?` and not `string`?

Reply:
I might not understand this correctly so I'm asking you:
Is the question why I used the nullable operator or you are asking me if I know to use it?
The reason it is there it's because I auto implemented the constructor in the IDE and this the Exception class under the hood.
 `
  public Exception(string? message, Exception? innerException)
            : this()
        {
            _message = message;
            _innerException = innerException;
        }
`